atlas

```diff
@@ -12,13 +12,12 @@ parser_definition() {
 }
 
 parser_definition_pull() {
-	setup   REST help:usage_pull -- \
-		"Usage: atlas pull [options...] [arguments...]"
+	setup   PULL_REST help:usage_pull -- \
+		"Usage: atlas pull [options...] [directories...]"
```

Comment:
I'm not sure the best way to ensure people know that they need to pass in directory mappings instead of just directories here. Maybe just using `directory mappings...` instead of `directories...`?

That's just a quick idea, I'm very open to any other wording that might better communicate this.

Reply:
Sure brian! Any improvement is welcome, especially when it comes to doc.

```suggestion
		"Usage: atlas pull [options...] [directory mappings...]"
```